web/ui/static/css/config.css

```diff
@@ -0,0 +1,3 @@
+.btn {
+    border-radius: 0;
+}
```

Comment:
Please end text files with a newline (same below).

Reply:
resolved in 9349188

Reply:
Hmm, now the files end with two newlines. I just meant not removing the trailing newline that should be there, as is standard in Unix/POSIX text files, see: https://stackoverflow.com/a/729795

All common editors will have a configuration setting to not remove the final newline character upon saving.